process/project_proposals.adoc

```diff
@@ -1,7 +1,92 @@
-== CNCF Project Proposal Process v1.2
+== CNCF Project Proposal Process v1.3
 
- . *Introduction*. This governance policy sets forth the proposal process for projects to be accepted into the Cloud Native Computing Foundation (“CNCF”). The process is the same for both existing projects which seek to move into the CNCF, and new projects to be formed within the CNCF.
- . *Project Proposal Requirements*. Projects must be proposed via https://github.com/cncf/toc/tree/master/proposals[GitHub]. Project proposals submitted to the CNCF (see https://github.com/cncf/toc/blob/master/proposals/kubernetes.adoc[example]) can be written in https://www.markdownguide.org[Markdown], http://asciidoc.org[AsciiDoc], or http://docutils.sourceforge.net/rst.html[reStructuredText] and must provide the following information to the best of your ability:
+=== Introduction 
+
+This governance policy sets forth the proposal process for projects to be accepted into the Cloud Native Computing Foundation (“CNCF”). The process is the same for both existing projects which seek to move into the CNCF, and new projects to be formed within the CNCF.
+
+=== Project stages
+
+image::project-stages.png[Project stages]
+
+=== Sandbox process
+
+All exceptions (and "no" outcomes) are handled by the TOC. Possible "no" outcomes include "not at this time", and the project may be encouraged to re-apply after addressing issues. Timeframes are approximate, to set expectations.
+
+image::sandbox-process.png[Sandbox process]
+
+. *Project Proposal*
+   * Project proposed through a https://github.com/cncf/toc/issues[GitHub issue]
+   * Template TBD
+. *TOC Triage* _2 weeks max_
```

Comment:
2 week max for triage phase is pretty tight. now the pipeline is > 10 weeks. saying "yes" to projects and moving projects to the next phase are straight forward. but if that is not the case, then 2 weeks is not enough since it will involve more discussion between TOCs and the community .

Reply:
Thanks @xiang90, I have clarified